CHANGES

```diff
@@ -11,63 +18,109 @@ Release 1.15 (development)
   by the ``system-info`` handler.
   See the :ref:`documentation for details <send-headers>`.
 * Support sending a ``RAUC-System-Version`` HTTP header in the initial
-  streaming/inspect request.
+  streaming/inspect request. (in :pr:`1654`)
 * Set up group memberships of the streaming helper process to allow access to
-  required resources (such as a mTLS private key). (by Omri Sarig)
+  required resources (such as a mTLS private key). (by Omri Sarig in :pr:`1695`)
 * Improve booted slot detection on UEFI systems by reading the ``BootCurrent``
-  variable. (by Pavel Löbl)
+  variable. (by Pavel Löbl in :pr:`1696`)
 * Refactor boot slot detection to prioritize explicit information from the
-  bootloader backend above the ``root=`` kernel commandline parameter.
+  bootloader backend above the ``root=`` kernel commandline parameter. (in
+  :pr:`1712`)
 * Introduce the ``allowed-signer-cns`` configuration option to restrict which
   certificates are accepted when verifying bundle signatures using a list of
-  CommonNames. (by Lars Schmidt)
+  CommonNames. (by Lars Schmidt in :pr:`1710`)
 * Allow (non-hidden) directories as bundle contents.
-  Hidden directories are reserved for internal use by RAUC. (by Jens Kehne)
+  Hidden directories are reserved for internal use by RAUC. (by Jens Kehne in
+  :pr:`1625`)
 * Produce progress updates for adaptive updates during block-hash-index
-  generation.
+  generation. (in :pr:`1734`)
 * Support a ``size-limit`` configuration option for the ``boot-emmc`` slot
   type.
   This can be used in cases where unrelated data on the eMMC boot partitions
   must be preserved and migrating it to a dedicated location is not feasible.
   Also ensure that the image fits into the slot before attempting to write it.
+  (in :pr:`1740`)
+* Improve handling of A/B/C setups by selecting the slot group with the oldest
+  installation date. (in :pr:`1525`)
```

Comment:
Should we mention that this also supports bootstrapping both slots' content from an external/factory system?

Reply:
Decided that this is more a pleasant side-effect, but not something we should recommend.